### PR TITLE
Add classification tutorial

### PR DESCRIPTION
**Pull request by sungmanc**
### Summary

The same with the title

### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have added e2e tests for validation.
- [ ] I have added the description of my changes into CHANGELOG in my target branch (e.g., [CHANGELOG](https://github.com/openvinotoolkit/training_extensions/blob/develop/CHANGELOG.md) in develop).​
- [ ] I have updated the documentation in my target branch accordingly (e.g., [documentation](https://github.com/openvinotoolkit/training_extensions/tree/develop/docs) in develop).
- [ ] I have [linked related issues](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

### License

- [ ] I submit _my code changes_ under the same [Apache License](https://github.com/openvinotoolkit/training_extensions/blob/develop/LICENSE) that covers the project.
      Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2024 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```

### CODE REVIEW

File: docs/source/guide/tutorials/base/how_to_train/classification.rst [hunk 1]
```diff
@@ -133,132 +175,367 @@ Let's prepare an OpenVINO™ Training Extensions classification workspace runnin
...
+
+        .. code-block:: shell
+
+            (otx) ...$ otx train --data_root data/flower_photos
+
+    .. tab-item:: CLI (with config)
+
+        .. code-block:: shell
+
+            (otx) ...$ otx train --config src/otx/recipe/classification/multi_class_cls/otx_mobilenet_v3_large.yaml --data_root data/wgisd
```
Comment (harimkang) [inline docs/source/guide/tutorials/base/how_to_train/classification.rst]:
```suggestion
            (otx) ...$ otx train --config src/otx/recipe/classification/multi_class_cls/otx_mobilenet_v3_large.yaml --data_root data/flower_photos
```

**Review by harimkang — DISMISSED** [review]
Thanks! LGTM

**Review by harimkang — APPROVED** [review]

**Review by jaegukhyun — APPROVED** [review]

File: docs/source/guide/tutorials/base/how_to_train/classification.rst [hunk 2]
```diff
@@ -133,132 +175,367 @@ Let's prepare an OpenVINO™ Training Extensions classification workspace runnin
...
+            |   └── epoch_*.pth
+            ├── tensorboard/
+            └── configs.yaml
+        └── .latest
+            └── train/
+    ...
+
+The training time highly relies on the hardware characteristics, for example on 1 NVIDIA GeForce RTX 3090 the training took about 3 minutes.
+
+After that, we have the PyTorch multi-class classification model trained with OpenVINO™ Training Extensions, which we can use for evaliation, export, optimization and deployment.
```
Comment (sungchul1) [inline docs/source/guide/tutorials/base/how_to_train/classification.rst:297]:
```suggestion
After that, we have the PyTorch multi-class classification model trained with OpenVINO™ Training Extensions, which we can use for evaluation, export, optimization and deployment.
```

File: docs/source/guide/tutorials/base/how_to_train/classification.rst [hunk 3]
```diff
@@ -133,132 +175,367 @@ Let's prepare an OpenVINO™ Training Extensions classification workspace runnin
...
-  Performance(score: 0.9659400544959128, dashboard: (3 metric groups))
 
 *********
 Export
 *********
 
 1. ``otx export`` exports a trained Pytorch `.pth` model to the OpenVINO™ Intermediate Representation (IR) format.
-It allows running the model on the Intel hardware much more efficient, especially on the CPU. Also, the resulting IR model is required to run PTQ optimization. IR model consists of 2 files: ``openvino.xml`` for weights and ``openvino.bin`` for architecture.
+It allows to efficiently run it on Intel hardware, especially on CPU, using OpenVINO™ runtime.
+Also, the resulting IR model is required to run PTQ optimization in the section below. IR model contains 2 files: ``exported_model.xml`` for weights and ``exported_model.bin`` for architecture.
```
Comment (sungchul1) [inline docs/source/guide/tutorials/base/how_to_train/classification.rst:360]:
```suggestion
Also, the resulting IR model is required to run PTQ optimization in the section below. IR model contains 2 files: ``exported_model.xml`` for architecture and ``exported_model.bin`` for weights.
```

File: docs/source/guide/tutorials/base/how_to_train/classification.rst [hunk 4]
```diff
@@ -133,132 +175,367 @@ Let's prepare an OpenVINO™ Training Extensions classification workspace runnin
...
-  (otx) ...$ otx optimize --load-weights openvino_model/openvino.xml \
-                          --output ptq_model
+        .. code-block:: python
 
-  ...
+            ckpt_path = "otx-workspace/20240312_052847/exported_model.xml"
+            engine.optimize(checkpoint=ckpt_path)
+
+
+The optimization time highly relies on the hardware characteristics, for example on 1 NVIDIA GeForce RTX 3090 it took about 10 minutes.
```
Comment (sungchul1) [inline docs/source/guide/tutorials/base/how_to_train/classification.rst:508]: Optimization doesn't use GPU.

**Review by sungchul1 — COMMENTED** [review]
There are some typos or inaccurate information. They're needed to be fixed.

**Pull request merged**